README.md

```diff
@@ -278,6 +278,43 @@ manual folds from the syntax generated ones. More information about this
 problem and on how to configure the addon can be found on its GitHub
 page.
 
+## FAQ
+
+### How do enable/disable indenting in modules?
+There are two config variables for this. `b:verilog_indent_modules` (which will
+be eventually deprecated) and the newer `g:verilog_disable_indent`. Because of
+the existence of `b:verilog_indent_modules`, indenting in modules is disabled by
+default. As well as this, because `b:verilog_indent_modules` is a buffer
+vairable, to ensure indenting in modules is always enabled you must add
+the following to your .vimrc:
+```VimL
+    augroup systemverilog_settings_1
+        au!
+        au Filetype verilog_systemverilog let b:verilog_indent_modules = 1
+    augroup END
+```
+The same also applies for `b:verilog_indent_preproc` (indentation of
+preprocessor statements).
+
+### Why is opening verilog/systemverilog files so slow?
```

Comment:
But if there's no way around it, maybe we could include the code you shared in the plugin, together with a variable to enable it and another to determine the maximum number of lines where folding is still enabled?